docs/Custom_Operation_for_GPUs.md

```diff
@@ -1123,7 +1124,7 @@ const T *UnpackDescriptor(const char *opaque, std::size_t opaque_len) {
 #endif
 ```
 
-#### `gpu_ops/kernels.h`
```

Comment:
My reading is that three headings are meant to appear under the `Test the backward function` heading, so we should keep them at fourth-level